src/ImageSharp/Processing/ResizeHelper.cs

```diff
@@ -337,21 +337,19 @@ private static (Size, Rectangle) CalculateMinRectangle(Size source, ResizeOption
             float percentHeight = MathF.Abs(height / (float)sourceHeight);
             float percentWidth = MathF.Abs(width / (float)sourceWidth);
 
-            float sourceRatio = (float)sourceHeight / sourceWidth;
-
             // Find the shortest distance to go.
             int widthDiff = sourceWidth - width;
             int heightDiff = sourceHeight - height;
 
             if (widthDiff < heightDiff)
             {
-                destinationHeight = (int)MathF.Round(width * sourceRatio);
+                destinationHeight = (int)MathF.Round(width * ((float)sourceHeight / sourceWidth));
                 height = destinationHeight;
                 destinationWidth = width;
             }
             else if (widthDiff > heightDiff)
             {
-                destinationWidth = (int)MathF.Round(height / sourceRatio);
+                destinationWidth = (int)MathF.Round(height * ((float)sourceWidth / sourceHeight));
```

Comment:
I think I prefer the old method. What are you fixing here?

Reply:
I agree this is a very minor performance improvement on probably less readability.
sourceRatio is calculated always, but only used when widthDiff != heightDiff (the third case doesn't use sourceRatio).
Inlining improves that special case, while the other cases are not affected negatively.

If it's not wanted I'm fine with undoing this again ;)

Reply:
It is indeed a tiny optimization. Maybe put the division in a variable inside to make it more clear that this is the aspect ration? You can also move `percentHeight` and `percentWidth`. These are only used at one spot.

Reply:
done.